frontend/document/koptinterface.lua

```diff
@@ -71,7 +88,7 @@ function KoptInterface:createContext(doc, pageno, bbox)
     -- Now koptcontext keeps track of its dst bitmap reflowed by libk2pdfopt.
     -- So there is no need to check background context when creating new context.
     local kc = KOPTContext.new()
-    local screen_size = Screen:getSize()
+    local screen_size = CanvasContext:getSize()
```

Comment:
I guess for coherent abstraction, stuff like `screen_size` should be named `canvas_size` (but that may add much noise to the diffs.)